go/felix/intdataplane/int_dataplane.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
```

Comment:
@fasaxc An unfortunate artefact here - should I squash and repush all this PR, so as to eliminate this change?